### DISCUSSION TITLE
Release v1.1.0 / 5.56.0 (pre-release)

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#1884] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

This is an experimental release it changes how sandboxie marks deleted host files as deleted, instead of creating dummy entries it uses an internal data structure to keep the stat of changes to the host file system. With the new mechanics it is possible to rename folders and keys properly.
This new behavior needs to be enabled by specifying UseFileDeleteV2=y sand UseRegDeleteV2=y in later builds this will be enabled by default, hence please test this pre release build and report back if there are any issues.

This build also adds a new UI functionality, sandman.exe now monitors the size of sandboxes and reports it in the info column, this functionality needs to be enabled in the global settings.

Last but not least this build reduces the cpu usage of sandman.exe a lot.

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

# ChangeLog

### Added
- added support for NtRenameKey (this requires UseRegDeleteV2=y) [#205](https://github.com/sandboxie-plus/Sandboxie/issues/205)
- added box size info [#1780](https://github.com/sandboxie-plus/Sandboxie/issues/1780)

### Changed
- reworked the mechanism sandboxie uses to mark host files as deleted
-- the new behavioure creates a data file in the box root FilePaths.dat instead of creating dummy files
-- it can be enabled with UseFileDeleteV2=y sane for the registry UseRegDeleteV2=y using RegPaths.dat
- reworked the TlsNameBuffer mechanism to be more versatile and less error prone
- significantly reduced the CPU usage of sandman.exe

### Fixed
- fixed folder rename issues (this requires UseFileDeleteV2=y) [#71](https://github.com/sandboxie-plus/Sandboxie/issues/71)
- fixed issue with process access [#1603](https://github.com/sandboxie-plus/Sandboxie/issues/1603)
- fixed translation issue [#1864](https://github.com/sandboxie-plus/Sandboxie/issues/1864)
- fixed ui issue with box selection window [#1867](https://github.com/sandboxie-plus/Sandboxie/issues/1867)
- fixed ui issue when switching language [#1871](https://github.com/sandboxie-plus/Sandboxie/issues/1871)

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/1.1.0'>Release v1.1.0 / 5.56.0</a>.</em>

**Comment by BilleBarrett** [comment]
I have a situation where a client is using a dashboard for his business, he is trying to get information let's say from 5/1 to 5/20
The dashboard compiles the info and then he clicks on export, but before Sandboxie can recover it, A Microsoft office window opens and sits there hung try to open windows to display the info.
Now if I run Chrome and the dashboard outside Sandboxie everything works ok, Is there a setting that is not set to help fix this problem
Thanks for the help
Bill

**Reply by DavidXanatos** [reply]
Is this issue present only in build 1.1.0 or also in prior builds?

**Reply by BilleBarrett** [reply]
I do not think this has anything to do with the new update
The dashboard is exporting a file Export_Client.xls for some reason Office 2019 starts to open and hangs there
It may be connected to Click_to_start program, I do not know
Bill
PS. I was able to download excel files from my PayPal Account as well as OneDrive
There must be something different that this online dashboard is doing, I'm going to try and see what it is doig

**Comment by BilleBarrett** [comment]
Question Why after I have Adobe Acrobat reader-selected in the App Templates it does not Sandbox PDFs
But if I go to force programs and put it in, it works, have to add the process and the folder
Just asking

**Comment by BilleBarrett** [comment]
The dashboard is exporting a file Export_Client.xls for some reason Office 2019 starts to open and hangs there
May be connected to Click to start program, I do not know

Bill

 

Remote Desktop Services
517 Miltwood Dr,  Greensboro, NC 27455-2709
(336) 988-0891    ***@***.***> ***@***.*** 
To subscribe to my FYI emails send an email to
 ***@***.***> ***@***.***
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’

**Comment by BilleBarrett** [comment]
I will observe what he is doing this morning
I just updated him, and before it finished it showed the 2 files that needed to be recovered 



 

Remote Desktop Services
517 Miltwood Dr,  Greensboro, NC 27455-2709
(336) 988-0891    ***@***.***> ***@***.*** 
To subscribe to my FYI emails send an email to
 ***@***.***> ***@***.***
Life is not a journey to the grave with the intention of arriving safely in a pretty and well-preserved body, but rather to skid in broadside, thoroughly used up, totally worn out,
and loudly proclaiming... 
 'Wow! What a ride!’
